packages/ava/src/utils/dataFormat.ts

```diff
@@ -21,5 +21,5 @@ export default function dataFormat(value: number | string, digits: number = 2) {
     .find((item) => {
       return value >= item.value;
     });
-  return item ? (value / item.value).toFixed(digits).replace(rx, '$1') + item.symbol : '0';
+  return item ? (value / item.value).toFixed(digits).replace(rx, '$1') + item.symbol : value.toFixed(digits);
```

Comment:
![image](https://user-images.githubusercontent.com/15177000/227204894-fd7fc608-6381-4d82-b952-8295b74cff7b.png)
#558

Reply:
I think it is acceptable to use `toFixed` in formatter function? But we indeed need a better data auto format util in `ava`, may do it in future PR ?

Reply:
aggre with it, precision issue is existed, but we could sovle it by a better format util in future